ui/helpers/utils/optimism/buildUnserializedTransaction.test.js

```diff
@@ -14,15 +13,13 @@ describe('buildUnserializedTransaction', () => {
         data: '0x0',
       },
     });
-    expect(unserializedTransaction).toMatchObject({
-      nonce: new BN('00', 16),
-      gasPrice: new BN('64', 16),
-      gasLimit: new BN('5208', 16),
-      to: expect.objectContaining({
-        buf: Buffer.from(times(20, 0)),
-      }),
-      value: new BN('09184e72a000', 16),
-      data: Buffer.from([0]),
+    expect(unserializedTransaction.toJSON()).toMatchObject({
```

Comment:
What motivated this change? Has `unserializedTransaction` changed with a dependency update?

Reply:
maybe some BN version mismatch? it wasn't obvious why, but the BNs did not match each other any more

Reply:
Ahh I see. So that's why `toJSON` is used, to stringify the numbers.